src/Avatar/Avatar.stories.tsx

```diff
@@ -27,10 +28,12 @@ storiesOf('Components/Avatar', module)
     .add('Default', () => (
         <ThemeProvider theme={RootTheme}>
             <StyledStory>
-                <p>Avatar</p>
+                <Typography tag="h1">Avatar</Typography>
                 <Avatar />
-                <p>Avatar With Initials</p>
+                <Typography tag="h1">Avatar With Initials</Typography>
                 <Avatar label={text('Avatar Initials', 'CC')} />
+                <Typography tag="h1">Avatar With Picture</Typography>
+                <Avatar src="https://avatars3.githubusercontent.com/u/24923730?s=460&v=4" />
```

Comment:
Might be a good idea to throw a knob in here for `src`

Reply:
👍